src/ReactiveUI/Cocoa/ReactiveNSViewController.cs

```diff
@@ -31,10 +20,7 @@ namespace ReactiveUI
     /// (i.e. you can call RaiseAndSetIfChanged)
     /// </summary>
     public class ReactiveViewController : NSViewController,
-        IReactiveNotifyPropertyChanged<ReactiveViewController>, IHandleObservableErrors, IReactiveObject
-#if UIKIT
```

Comment:
Yeah I checked whether or not this should be able to work, and all the functionality seems to be there actually. So I didn't see a reason to leave the check for `UIKIT` there.